unbound/USAGE-server.md

```diff
@@ -0,0 +1,23 @@
+
+# Usage of the DNS server
+
+[Unbound DNS server](https://github.com/NLnetLabs/unbound) was configured with tls connection using the post quantum [openssl](https://github.com/open-quantum-safe/openssl) variant.
+## Installation
+Assuming you have docker [installed](https://docs.docker.com/install) on your machine all command below will launch dns server docker.
+
+Run Unbound DNS container:
+```bash
+    docker network create unbound-test
+    docker run --network unbound-test --interactive --publish=853:853 --tty --hostname unbound --name unbound -it openquantumsafe/unbound
+```
+After running all the command above a container will open with unbound running configure with dns-over-tls. 
+Before running unbound, a certificate is needed. Therefore a self sign certificate will be generate and ask for input. 
+
+The key exchange between the server and the client is set on p384_kyber768:X25519, other key exchange algorithms can be used, find more algorithm in the [list of available post quantum key exchange algorithms](https://github.com/open-quantum-safe/boringssl#key-exchange). To specify the desire key exchange algorithm use the parameter -e in the docker run command such below example wherre kyber1024 is used to do key exchange.
```

Comment:
nit/typo: "desire" -> "desired". Also maybe quote `docker run` to see it suitably highlighted.